<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkRemoteSettingsPage</class>
 <widget class="QWidget" name="NetworkRemoteSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Remote</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="use_remote">
     <property name="text">
      <string>Use a network remote control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="use_remote_container_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_remote_port">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="remote_port">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="only_non_public_ip">
        <property name="toolTip">
         <string>Only accept connections from clients within the ip ranges:
10.x.x.x
172.16.0.0 - 172.31.255.255
192.168.x.x</string>
        </property>
        <property name="text">
         <string>Only allow connections from the local network</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="use_auth_code">
        <property name="toolTip">
         <string>A client can connect only, if the correct code was entered.</string>
        </property>
        <property name="text">
         <string>Require authentication code</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="auth_code">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>Enter this IP in the App to connect to Clementine.</string>
        </property>
        <property name="text">
         <string>Your IP address:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="ip_address">
        <property name="text">
         <string notr="true">127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="allow_downloads">
        <property name="toolTip">
         <string>Allow a client to download music from this computer.</string>
        </property>
        <property name="text">
         <string>Allow downloads</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QGroupBox" name="download_settings_container">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Download settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="convert_lossless">
           <property name="toolTip">
            <string>Convert lossless audiofiles before sending them to the remote.</string>
           </property>
           <property name="text">
            <string>Convert lossless files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="format_container">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Audio format</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QPushButton" name="options">
              <property name="text">
               <string>Options...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="format"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Root folder that will be browsable from the network remote</string>
        </property>
        <property name="text">
         <string>Files root folder</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="FileChooserWidget" name="files_root_folder" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="files_music_extensions"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>coma separated list of the allowed extensions that will be visible from the network remote (ex: m3u,mp3,flac,ogg,wav)</string>
        </property>
        <property name="text">
         <string>Music extensions remotely visible</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Download the remote for Desktops, Android and iOS</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="desktop_remote">
        <property name="text">
         <string>desktop_remote</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="play_store_2">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>or scan the QR code: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="android_app_qr_2">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/data.qrc">:/clem_remote_android_qr.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="apple_store">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/data.qrc">:/en_available_apple_store_45.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>or scan the QR code: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iOS_app_qr">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/data.qrc">:/clem_remote_iOS_qr.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Download the original Android app</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="play_store">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>63</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>or scan the QR code: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="android_app_qr">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="../../data/data.qrc">:/clementine_remote_qr.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>98</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooserWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/filechooserwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>use_auth_code</sender>
   <signal>toggled(bool)</signal>
   <receiver>auth_code</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_remote</sender>
   <signal>toggled(bool)</signal>
   <receiver>use_remote_container_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allow_downloads</sender>
   <signal>toggled(bool)</signal>
   <receiver>download_settings_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>convert_lossless</sender>
   <signal>toggled(bool)</signal>
   <receiver>format_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
